tests/client-tests.js

```diff
@@ -50,26 +67,62 @@ describe("Hull", () => {
         .that.eql("123456");
     });
 
-    it("should allow to pass account id as an object property", () => {
+    it("should allow to pass account name as an object property", () => {
       const hull = new Hull({ id: "562123b470df84b740000042", secret: "1234", organization: "test" });
 
-      const scoped = hull.asAccount({ id: "123456" });
+      const scoped = hull.asAccount({ name: "Hull" });
       const scopedConfig = scoped.configuration();
       const scopedJwtClaims = jwt.decode(scopedConfig.accessToken, scopedConfig.secret);
       expect(scopedJwtClaims)
-        .to.have.property("sub")
-        .that.eql("123456");
+        .to.have.property("io.hull.asAccount")
+        .that.eql({ name: "Hull" });
     });
 
```

Comment:
We have one test for `hull.asAccount()` without user resolution. I would test `traits` on other place, but thanks, for pointing it out.